classes/class.ilObjORCAContentGUI.php

```diff
@@ -346,6 +357,7 @@ function get_orca_tools()
         }
     }
 
+   
```

Comment:
please do not add non-needed empty lines. Max. empty lines between methods is 2, usually 1.